S23/sandeep-1-kumar/LowerCasing.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Locale;
+
+public class LowerCasing {
+
+    public static void main(String[] args)
+    {
+
+    }
+    public String lowerCaseConversion(String languageType,String word)
+    {
+        String[] exceptedLanguages = {"zh" , "ja" , "th"};
+        Locale locale = Locale.forLanguageTag(languageType);
+        String language = locale.getLanguage();  
+        for(String element : exceptedLanguages)
+        {
+            if(element.equals(language))
+            {
+                return word;
+            }
+        }
+        String returningValue = word;
+        if(language.equals("tr") || language.equals("az"))
+        {
+            returningValue = word.replace('I' ,'\u0131');
+        }
+        else if(language.equals("el"))
+        {
+            if (word.endsWith("Σ"))
```

Comment:
ΠΌΛΗΣΠΌΛΗΣ el-GR ---> πόλησπόλης
this test case fails

Reply:
![img-1](https://user-images.githubusercontent.com/118848198/220710066-f5f41c98-35b4-46f1-b6cb-2f1cdf9a88d3.jpeg)

I can clearly see that the response is fine please check the screen shot and confirm.

Reply:
here, I hope check is implemented for lang "el" but not for all lang belonging to type "el", i.e "el-GR"